coalib/core/Bear.py

```diff
@@ -0,0 +1,502 @@
+from functools import partial
+import inspect
+import logging
+from os import makedirs
+from os.path import join, abspath, exists
+
+from appdirs import user_data_dir
+
+from coala_utils.decorators import (enforce_signature, classproperty,
+                                    get_public_members)
+
+import requests
+
+from coalib.results.Result import Result
+from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.FunctionMetadata import FunctionMetadata
+from coalib.settings.Section import Section
+
+
+class Bear:
+    """
```

Comment:
so this documentation comment  is helpful to me. But not to somebody wanting to know what a bear is and what he has to look out for when implementing one - this should rather be a sort of tutorial for bears *or* at first link to one.